bedrock/firefox/views.py

```diff
@@ -529,7 +529,8 @@ def get_redirect_url(self, *args, **kwargs):
 class WhatsnewView(L10nTemplateView):
 
     ftl_files_map = {
-        'firefox/whatsnew/index-account.html': ['firefox/whatsnew/whatsnew-account', 'firefox/whatsnew/whatsnew']
+        'firefox/whatsnew/index-account.html': ['firefox/whatsnew/whatsnew-account', 'firefox/whatsnew/whatsnew'],
+        'firefox/whatsnew/whatsnew-fx79.html': ['send_to_device', 'firefox/whatsnew/whatsnew', 'firefox/whatsnew/whatsnew-fx79']
```

Comment:
The .ftl file `send_to_device` is now a global file, so no need to include it here. 

Much the same as .lang, the order of these files can also govern locale activation. The file `firefox/whatsnew/whatsnew-fx79` should be first, before `firefox/whatsnew/whatsnew`.